docs/guides/migrations.md

```diff
@@ -0,0 +1,83 @@
+<!--[meta]
+section: guides
+title: Migrations
+[meta]-->
+
+## Migrations in Keystone
+
+Migrations are required when you update a List in Keystone after the database has been initialised. **Keystone can't automatically manage these migrations for you** because it doesn't know what to do with the existing data. This guide will explain when and why you need database migrations and offer some basic solutions to get you started. As your project grows to include production deployments, how you manage the seeding and migration of data, will depend on your specific situation. Keystone tries not to limit type of applications you build and therefor cannot offer a "one size fits all" solution to migrations.
```

Comment:
I mean,  the schema's are declarative,  you probably could produce a reasonable migration output,  but,  that effort probably isn't worth it

Reply:
Yeah, that's something we've tossed around as an idea. It falls in the category of PRs welcome for now, but maybe something we look at down the line once we have a better understanding of the different use cases and problems we're trying to solve for